app/controllers/applications_controller.rb

```diff
@@ -143,7 +106,7 @@ def set_applicant_id
     end
   end
 
-  def skip_validation
-    @application.skip_validation = true
+  def guest
```

Comment:
I don't understand this method. Does the guest parameter signify that I am a guest? Isn't `current_user.blank?` the same information?

Reply:
@A5308Y Is not: we want to render :continue_as_guest if the user is not logged in, but also allow them to create an application as guest if they chose to do so. That means that we need a way to identify those users who are guests. We were doing it with commit param messages, and I thought this could be a better way to do it. Do you have another idea?

Reply:
In all apps I've worked on we used devise.  Devise sets `current_user` when I enter my password and email correctly. Otherwise I'm not logged in. I would call not-logged-in users "guests". They could then still create applications. Is your setup different?

Reply:
Yes, is the same set up but defined by Clearance not Devise (I think). current_user returns the user object if they are logged in. If the user is not logged-in, current_user returns nil. The problem is that we need a way to define the desire of users to not log in and allow them to create applications. So we need to differenciate between users that are just not logged in (and show them the reminder to log in/register or just continue as guest, AND those users who decided to continue as guests and not register/log in but still create an applications. This guest params defines the last case. If you have another idea on how could we do this (have both cases), please let me know, I would be very happy to learn that :)